package-parser/tests/data/boundaries/api_data.json

```diff
@@ -42,20 +42,20 @@
                     "is_public": true,
                     "assigned_by": "POSITION_OR_NAME",
                     "docstring": {
-                        "type": "int in range (0, infinity)",
-                        "description": ""
+                        "type": "",
+                        "description": "int in range (0, infinity)"
                     }
                 },
                 {
-                    "name": "parameter_with_upper_bound",
+                    "name": "parameter_with_upper_bound_and_boundary",
```

Comment:
Is this meant to say `parameter_with_upper_bound_and_union`? In any case, the `qname`, `pname`, and the information in the `boundaries/annotation_data.json` file should also be updated.